spec/certificate_request_spec.rb

```diff
@@ -1,22 +1,20 @@
 require 'spec_helper'
 
 describe Acme::CertificateRequest do
-  before(:all) do
-    @test_key = generate_private_key
-  end
+  let(:test_key) { generate_private_key }
```

Comment:
This seem to indeed make the specs on average 3 (2-4) seconds slower on my machine.